SYSTEMS.md

```diff
@@ -193,3 +303,28 @@ Code:
 Result:
 
 ![Basic Example](doc_images/ex1.png)
+
+## **Notes for Forge Users:**
+  * It is best to use native Foundry for testing until you are 100% certain you are happy with your template. 
+  * Between The Forge's upload system, where if it sees a file exists, it won't upload it again, and browsers caching files, you will find that if you upload a changed .json, it will appear the same as before the edits. This is why testing is best on Foundry where you can just save the file, then F5 your game, and uploads are instantaneous. It is highly recommended that multiple edited uploads be a different file name every time, to avoid this issue.
```

Comment:
```suggestion
  * Between The Forge's upload system that will not upload a file again if it sees the file already exists, and various browsers caching files, you may find that if you upload a changed .json, the changes will not appear. This is why testing is best on Foundry where you can just save the file, then refresh your client (F5), and changes should be instantaneous. If you can only use Forge then It is highly recommended that multiple edited uploads be a different file name every time, to avoid this issue, until your template is complete.
```